legal-api/src/legal_api/services/involuntary_dissolution.py

```diff
@@ -47,15 +47,15 @@ class EligibilityDetails:
 
     @classmethod
     def check_business_eligibility(
-        cls, identifier: str, exclude_in_dissolution=True
+        cls, identifier: str, exclude_in_dissolution=True, exclude_future_effective_filing=False
```

Comment:
we had talked about things starting to get messy with the exclude parameters and to use an enum to encapsulate `exclude_in_dissolution` as well as `exclude_future_effective_filing`.  

Can we do that?

Reply:
I did